scripts/docker-ls.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -e
+
+# Find all directories with a package.json file in /src (excluding node_modules)
+directories=$(find src -type d -name "node_modules" -prune -o -type f -name "package.json" -exec dirname {} \; | sort)
+
+for dir in $directories; do
+	(
+		echo "COPY $dir/package.json $dir/"
+	)
+done
```

Comment:
@alexey-yarmosh I removed the priority for `extensions/lib` since we copy all files first, then build, it should be ok, I think.